src/screens/FeedScreen/Header.js

```diff
@@ -537,13 +537,13 @@ const styles = StyleSheet.create({
     flexDirection: 'row'
   },
   textFollow: {
-    color: COLORS.blue,
+    color: colors.bluePrimary,
     fontSize: normalizeFontSize(14),
     fontStyle: 'normal',
     fontWeight: '500'
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values have been updated to use the new `colors` import. Ensure that these new color values are equivalent to the old ones and that they match the intended design.
```diff
-    color: COLORS.blue,
+    color: colors.bluePrimary,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->